app/views/subscriptions/_course.html.erb

```diff
@@ -0,0 +1,18 @@
+<% if @courses.empty? %>
+  <p>Nenhum curso encontrado com esse nome</p>
+<% else %>
+  <p><strong>Foram encontrados <%= @courses.size %> cursos: </strong></p>
+  <% @courses.each do |course| %>
+    <hr>  
+    <div id="CURSO<%= course.id %>">
+      <p><%= course.name %></p>  
+      <p><%= course.description %></p>
+
+      <%= form_with url: add_course_subscription_path do |form| %>
```

Comment:
Aqui acho que daria para fazer só um `<%= link_to "Adicionar Curso", add_course_subscripition_path( passando_course ), method: post %>`
=> https://stackoverflow.com/questions/13414663/using-rails-link-to-for-links-that-post  algo assim em um link.

Reply:
da não, é preciso passar o parametro do curso que vai ser adicionado... fiz uma gambiarra com hidden form, mas isso da pra ser ajustado de outra maneira

Reply:
Pra funcionar esse jeito via link você teria que mudar suas rotas pra algo tipo:

```
resources :subscriptions do
  resources :courses, only: [] do
    post :add, on: :member
  end
end
```

que aí o link ficaria algo como `add_course_subscription_path(@subscription, @course)` mas acho que você podia tentar algo assim. Trocar todo esse each para:

```
<%= form_with url: add_course_subscription_path(@subscription) do |form| %>
  <%= f.collection_select :course_id, @courses, :id, :name %>
  <%= f.submit 'Adicionar curso' %>
<% end %>
```

Porque aí esse curso ai invés de ir como um hidden field você manda o ID dele através de um select. Daria pra fazer via botão também mas ficaria meio estranho assim:

<%= link_to "Adicionar Curso", add_course_subscription_path(@subscription, course_id: course.id), method: post %>